src/ec/packages/ec-component-search-form/docs/search-form.md

```diff
@@ -0,0 +1,40 @@
+# ECL Twig - EC Search Form
+
+npm package: `@ecl-twig/ec-component-search-form`
+
+```shell
+npm install --save @ecl-twig/ec-component-search-form
+```
+
+## Search Form
+
+### Parameters
+
+- "textInput" (associative array) default: A predefined structure for EC Text Input
+- "button" (associative array) default: A predefined structure for EC Button
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the form
+- "extra_attributes" (optional) (array) (default: []) Extra attributes for the form
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+
+### Example:
+
+```twig
+{% include 'path/to/icon.html.twig' with {
```

Comment:
by adding two spaces at the end of the line, you should be able to force a new line. It would make the code more readable (currently it is a single very long line)

Reply:
I would like to understand better what you mean - I understand that there is a lack of breaking the line, but what is the exact line(s) in this case?

Reply:
I also dont get it for me it loks ok https://github.com/ec-europa/ecl-twig/blob/ec-component-search-form/src/ec/packages/ec-component-search-form/docs/search-form.md

Reply:
> by adding two spaces at the end of the line, you should be able to force a new line. It would make the code more readable (currently it is a single very long line)

Ok I understand - you mean displaying in the Storybook (Notes section). Fixed in https://github.com/ec-europa/ecl-twig/pull/28/commits/0d2dc7f309fc7f3c2fc9b78972b2451c58fa8a63